### PR TITLE
Favorites users

### PR DESCRIPTION
**Pull request by leshihak**

### CODE REVIEW

File: src/components/containers/UsersList/userList.styled.ts [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
+import { Box, List } from "@mui/material";
 import styled from "styled-components";
 
 export const StyledList = styled(List)`
   @media (max-width: 599px) {
     margin-right: unset;
   }
 `;
+
+export const StyledBox = styled(Box)`
```
Comment (tronya) [inline src/components/containers/UsersList/userList.styled.ts]: rename this variable, and color could be different from orange + variable, not declared directly

File: src/components/containers/UsersList/usersList.component.tsx [hunk 2]
```diff
@@ -77,11 +83,17 @@ export const UsersList: FC<ListWrapperProps> = ({
                   </Typography>
                 }
               />
+              {router.pathname === "/user/favorites" &&
```
Comment (tronya) [inline src/components/containers/UsersList/usersList.component.tsx]: Maybe its better to create additional property like f.e. showFavirites or similar for userList.component, and by default set to false.
`cause f.e. i we will use it somwhare else on another page we need to extend if statement.

File: pages/user/favorites.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  const [favorites, setFavorites] = useState<string[]>([]);
+
+  useEffect(() => {
+    getGeolocation()
+      .then((result) => setGeolocation(result))
+      .catch((error) => toast.error(error));
+  }, []);
+
+  useEffect(() => {
+    getFavorites()
```
Comment (tronya) [inline pages/user/favorites.tsx]:
```suggestion
    export const getFavoritesCollection = async (someCondition) =>
  await getDocs(query(
    collection(DB, "favorites"),
    where("id", "==", "djMgiqLbFfdqmDIXi0Oenkp1lPe2") // this is that some condition
  ));
```

File: src/components/containers/UsersList/usersList.component.tsx (hunk 2 above)
Comment (leshihak) [reply src/components/containers/UsersList/usersList.component.tsx]: fixed it

**Pull request merged**